buildSrc/src/main/groovy/tanagra.java-conventions.gradle

```diff
@@ -21,29 +21,28 @@ plugins {
 repositories {
     mavenCentral()
     maven {
-        url "https://broadinstitute.jfrog.io/broadinstitute/libs-release/"
+        url 'https://broadinstitute.jfrog.io/broadinstitute/libs-release/'
     }
     maven {
-        url "https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/"
+        url 'https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/'
     }
     google()
 }
 
-sourceCompatibility = JavaVersion.VERSION_11
+sourceCompatibility = JavaVersion.VERSION_17
 
 // See https://docs.gradle.org/current/userguide/dependency_locking.html
 dependencyLocking {
     lockAllConfigurations()
-    lockMode = LockMode.STRICT
```

Comment:
Why the change from strict > lenient?

Reply:
This was due to spotless config errors  - with a strict mode every build command leads to below error 

```
Could not determine the dependencies of task ':underlay:spotlessCheck'.
> Could not create task ':underlay:spotlessGroovyGradleCheck'.
   > Could not create task ':underlay:spotlessGroovyGradle'.
      > Could not resolve all dependencies for configuration ':underlay:spotless-1757186549'.
         > Locking strict mode: Configuration ':underlay:spotless-1757186549' is locked but does not have lock state.
```

to fix it either the lock Mode must be DEFAULT or the '--write-locks ' option must be specified every time
```
 >> ./gradlew underlay:build --write-locks 
```

I looked into this a bit but could not figured the root cause, hence removed the STRICT mode, keeping it consistent with other Verily repo configs downstream

Reply:
Is the configuration name randomly generated? i.e. does 1757186549 change or is it consistent. On the off chance it's consistent, you might try just writing locks for that task. If not, removing strict sgtm.

Reply:
Manually updating the lock file is not recommended. As discussed leaving it with default lock mode (not strict)